src/build.py

```diff
@@ -103,7 +103,7 @@ def run_make(self):
         self.context.log("Dumping symbols from elf...")
         with open(elf_path_to_syms(self.context.elf), "w+", encoding="utf-8") as syms:
             result = subprocess.run([
-                "objdump",
+                "aarch64-none-elf-objdump",
```

Comment:
Does this change things?

Reply:
hey, sorry, this fork is kind of a mess rn and the listing isn't currently working. that line itself allows it to work on windows, where `objdump` doesn't exist by default. please don't merge this pr right now, i'll let you know when it's ready

Reply:
hmm this tool is kinda meant for linux only because windows is kinda weird

Reply:
maybe this can be made a config

Reply:
the tool works fine on windows aside from that particular part it seems, and `aarch64-none-elf-objdump` seems like a better option than `objdump` on linux as well and it should exist on both platforms as long as you have devkitA64